src/main/resources/google_checks.xml

```diff
@@ -336,7 +336,7 @@
       <property name="tokens" value="METHOD_DEF, CTOR_DEF, ANNOTATION_FIELD_DEF, COMPACT_CTOR_DEF"/>
     </module>
     <module name="MissingJavadocMethod">
-      <property name="scope" value="public"/>
+      <property name="scope" value="protected"/>
```

Comment:
Changes made based on following:

[7.3 Where Javadoc is used](https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required)

> At the minimum, Javadoc is present for every public class, and every public or protected member of such a class, with a few exceptions noted below.

We need to log violations for protected members too.

Reply:
We should not mix user affecting changes with internal migration/refactoring.
Please postpone this update to next PR.

Reply:
reverted this and also removed test cases related to this